support-frontend/assets/pages/paper-subscription-checkout/helpers/options.ts

```diff
@@ -20,8 +20,8 @@ import { getHomeDeliveryDays } from 'pages/paper-subscription-checkout/helpers/h
 import { getVoucherDays } from 'pages/paper-subscription-checkout/helpers/voucherDeliveryDays';
 
 function getProductOption(): PaperProductOptions {
-	const productInUrl = getQueryParameter('product');
-	// $FlowIgnore - flow doesn't recognise that we've checked the value of productInUrl
+	const productInUrl = getQueryParameter('product') as PaperProductOptions;
+
```

Comment:
Maybe to avoid the type assertion here we could write like

```ts
	const productInUrl = getQueryParameter('product');

	return (
		ActivePaperProductTypes.find((product) => product === productInUrl) ?? Everyday
	);
```

Reply:
I think we'd still need a type assertion as `getQueryParameter` returns a string

Reply:
Very good to know!